src/hooks/bin/configure

```diff
@@ -83,9 +89,34 @@ handle_php_memory_limit()
 	php_set_previous_memory_limit "$memory_limit"
 }
 
+handle_cronjob_interval()
+{
+	interval="$(cronjob_interval)"
+	previous_interval=$CRONJOB_INTERVAL
```

Comment:
From what I can see, this will never be defined. Is there a reason you didn't follow a model similar to that given in the other configurable items (saving the previously-set interval, etc.)?

Reply:
Not really. Thought it was simpler. Going to change it to model the other configurable items.